### DISCUSSION TITLE
NoSQL support?

### DISCUSSION THREAD

**Discussion by arcanosam** [sqlpage/SQLPage#698] [Ideas]
Hi all.

I hope everyone is ok.

It's just for my curiosity and I didn't find information here in the Discussion's area.

Is there a plan in future to support NoSQL?

Is something complex to include?

I'm not a developer nowdays, but I like very much this project and this idea come to me and that why I'm asking, just to know what you think about it

Thanks in advance and all the best for you all that make this project grow and possible

Samuel

------

Reviewing the Readme, I think I can enter in problem of misconception of mine with the purposes from this project.

But I still keep this discussion to intention of learning and see, maybe, where I'm wrong about this idea

Thanks for your understanding.

**Comment by lovasoa** [comment]
Hi @arcanosam !
  Adding support for other databases, including NewSQL databases, is on the roadmap. There are also plans to build user interfaces that allow making applications without having to interact with the sql code as text directly at all. However, at its core, SQLPage itself is quite deeply tied to the sql language. Building something similar to SQLPage for a NoSQL database would be possible, but that would be a different project.

**Reply by arcanosam** [reply]
Hi @lovasoa 

Thanks for your fast response.

 It's already satisfay my curiosity and either show me about newSQL project which I didn't know it.

It will be great too when having this functionality of CRUD maker.

That's why like to follow your project, always bringing great tools to developers

Cheers

Samuel
